util/stmtsummary/statement_summary_test.go

```diff
@@ -1008,114 +1005,116 @@ func (s *testStmtSummarySuite) TestSetMaxStmtCountParallel(c *C) {
 		// Add 32 times with different digest.
 		for i := 0; i < loops; i++ {
 			stmtExecInfo1.Digest = fmt.Sprintf("digest%d", i)
-			s.ssMap.AddStatement(stmtExecInfo1)
+			ssMap.AddStatement(stmtExecInfo1)
 		}
 	}
 	for i := 0; i < threads; i++ {
 		go addStmtFunc()
 	}
 
-	defer c.Assert(s.ssMap.SetMaxStmtCount("", true), IsNil)
+	defer require.Nil(t, ssMap.SetMaxStmtCount("", true))
```

Comment:
Arguments are evaluated immediately, maybe the original logic is wrong? I guess he want to `setmaxstmtcount` after the return.

Reply:
Oops, it is in the part1, I've commented in 26870, resolved.